docker/k8s/vtadmin/Dockerfile

```diff
@@ -40,16 +39,12 @@ COPY --from=k8s /vt/web/vtadmin /vt/web/vtadmin
 RUN npm --prefix /vt/web/vtadmin ci && \
     npm run --prefix /vt/web/vtadmin build
 
-# add vitess user/group and add permissions
-RUN deluser node && \
-   groupadd -r --gid 2000 vitess && \
-   useradd -r -g vitess --uid 1000 vitess && \
-   chown -R vitess:vitess /vt && \
-   chown -R vitess:vitess /vtdataroot
 
-USER vitess
+FROM nginxinc/nginx-unprivileged:1.22-alpine AS nginx
 
-VOLUME /vt/web/vtadmin/build/config
+ENV VTADMIN_WEB_PORT=14201
 
-CMD ["/bin/bash", "-c", "/vt/web/vtadmin/node_modules/.bin/serve --no-clipboard -l $VTADMIN_WEB_PORT -s /vt/web/vtadmin/build"]
+COPY --from=node /vt/web/vtadmin/build /var/www/
+COPY default.conf /etc/nginx/templates/default.conf.template
 
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
we don't need to redefine the CMD, it's already in the base image: https://github.com/nginxinc/docker-nginx-unprivileged/blob/main/stable/alpine/Dockerfile#L150